roles/kubernetes/kubeadm/tasks/kubeadm_etcd_node.yml

```diff
@@ -3,8 +3,13 @@
   command: >-
     {{ bin_dir }}/kubeadm init phase
     --config {{ kube_config_dir }}/kubeadm-config.yaml
-    upload-certs --experimental-upload-certs
-    {% if kubeadm_certificate_key is defined %}
+    upload-certs
+    {% if kubeadm_version is version('v1.15.0', '<') %}
+    --experimental-upload-certs
+    {% else %}
+    --upload-certs
+    {% endif %}
+    {% if kubeadm_certificate_key is defined and kubeadm_version is version('v1.15.0', '<') %}
     --certificate-key={{ kubeadm_certificate_key }}
```

Comment:
Is `--certificate-key` not required in versions after 1.15 ?

Reply:
Let me check the documentation, but the error message said:
`can not mix '--config' with arguments [certificate-key]`
I'll get back to this.

Reply:
It is not, because the certificate key is already included in `kubeadm-config.yaml`.

It's a bit nicer to nest the `if` statements, like done in `kubeadm-setup.yml`:
```diff
--- a/roles/kubernetes/kubeadm/tasks/kubeadm_etcd_node.yml
+++ b/roles/kubernetes/kubeadm/tasks/kubeadm_etcd_node.yml
@@ -1,12 +1,16 @@
 ---
 - name: Refresh certificates so they are fresh and not expired
   command: >-
-    {{ bin_dir }}/kubeadm init phase
+    {{ bin_dir }}/kubeadm init phase upload-certs
     --config {{ kube_config_dir }}/kubeadm-config.yaml
-    upload-certs --experimental-upload-certs
+    {% if kubeadm_version is version('v1.15.0', '<') %}
+    --experimental-upload-certs
     {% if kubeadm_certificate_key is defined %}
     --certificate-key={{ kubeadm_certificate_key }}
     {% endif %}
+    {% else %}
+    --upload-certs
+    {% endif %}
   register: kubeadm_upload_cert
   delegate_to: "{{ groups['kube-master'][0] }}"
   when: kubeadm_etcd_refresh_cert_key
```